### DISCUSSION TITLE
rewrite or internal redirection cycle while internally redirecting

### DISCUSSION THREAD

**Discussion by mchlrv** [Mailu/Mailu#3117] [User support]
## Environment & Version

- `docker compose version`: v2.21.0
- Version: `master`

## Description
After updating the server today it the latest master version I'm unable to reach webgui at the root ("/") level. The /webmail and /admin work fine but / gives an `500 Internal Server Error NGINX` errror. 

## Observed behaviour
My mailu.env looks like the config below. However I have also tried to change WEBROOT_REDIRECT to `WEBROOT_REDIRECT=/webmail` but no success.

```
# Path to redirect / to
WEBROOT_REDIRECT=/

# Path to the admin interface if enabled
WEB_ADMIN=/admin

# Path to the webmail if enabled
WEB_WEBMAIL=/webmail
```

## Logs
Checking at the docker compose logs I get:
```
mailu-front-1      | 2024/01/03 20:48:41 [error] 15#15: *155 rewrite or internal redirection cycle while internally redirecting to "/", server: , request: "GET / HTTP/1.1"
```

**Comment by mchlrv** [comment]
This report has been transferred to discussion by @nextgens. However it is not clear to me why? The previous install was working fine and all I did was docker compose pull and restarted docker. It seems it is not working as the documentation describes (or I might have missed something), so this should be an issue rather than an discussion right?

**Reply by nextgens** [reply]
There is nothing in what you wrote above that suggests that this is a problem requiring code changes in Mailu.

User support goes do discussions, those that understand what you mean may answer; I don't.

Please do not highlight me.

**Comment by ghostwheel42** [comment]
When using master you are using a development version.
A `docker compose pull` may not be everything you need to do to have it in a working condition.
If you do not want to run a development version yout should stickt to the released versions (currently 2.x).
You probably have to re-run the setup wizard for master to get your master version on track again.

Alex

**Reply by mchlrv** [reply]
I've been following the steps of the documentation and haven't specified any version. As far as I know in that case it will default to 2.0. So doing a `docker compose pull` should only download the updated version of version 2 right? Looking at the docker compose file the image is set to: `image: ${DOCKER_ORG:-ghcr.io/mailu}/${DOCKER_PREFIX:-}nginx:${MAILU_VERSION:-2.0}` in the case of nginx.

For now I have referred back my local images and it is working again so something clearly has changed.
